src/HttpResponse.cc

```diff
@@ -112,10 +112,12 @@ void HttpResponse::validateResponse() const
                      error_code::HTTP_PROTOCOL_ERROR);
 }
 
-std::string HttpResponse::determineFilename() const
+std::string HttpResponse::determineFilename(
+    bool content_disposition_default_utf8) const
```

Comment:
aria2 project prefers CamelCase for variables and function names.  Perhaps, contentDispositionUTF8?